packages/mazeppa/mazeppa.0.4.0/opam

```diff
@@ -0,0 +1,48 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A modern supercompiler for call-by-value functional languages"
+maintainer: ["Lewis Pearson <hirrolot@gmail.com>"]
+authors: ["Lewis Pearson <hirrolot@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/mazeppa-dev/mazeppa"
+doc: "https://github.com/mazeppa-dev/mazeppa"
+bug-reports: "https://github.com/mazeppa-dev/mazeppa/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "dune" {>= "3.14"}
+  "pprint"
+  "checked_oint" {>= "0.2.1"}
+  "ppx_deriving"
+  "ppx_string_interpolation"
+  "ppx_yojson_conv"
+  "ppx_blob" {>= "0.9.0"}
+  "bisect_ppx"
+  "menhir" {>= "20180523"}
```

Comment:
Ah, and we get one more lower bounds error now

```
#=== ERROR while compiling mazeppa.0.4.0 ======================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.4.14.2 | pinned(https://github.com/mazeppa-dev/mazeppa/archive/refs/tags/v0.4.0.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/mazeppa.0.4.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p mazeppa -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/mazeppa-7-9f5ad1.env
# output-file          ~/.opam/log/mazeppa-7-9f5ad1.out
### output ###
# File "lib/dune", line 22, characters 0-26:
# 22 | (menhir
# 23 |  (modules parser))
# Error: Rule failed to generate the following targets:
# - lib/parser.conflicts
```

I think indicating that we need at least [20180905](https://gitlab.inria.fr/fpottier/menhir/-/blob/master/CHANGES.md?ref_type=heads#20180905):

```suggestion
  "menhir" {>= "20180905"}
```